assets/controllers/form_upload_controller.js

```diff
@@ -0,0 +1,17 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+    static targets = ['input', 'form'];
+
+    connect() {
+        this.inputTarget.addEventListener('change', this.#submit);
+    }
+
+    open() {
+        this.inputTarget.click();
+    }
+
+    #submit = () => {
+        this.formTarget.submit();
+    }
```

Comment:
Grâce à la réutilisation de `form-submit` on n'utilise plus tout ce qui concerne le submit ici

```suggestion
    static targets = ['input'];

    open() {
        this.inputTarget.click();
    }
```

Reply:
Indeed!